rackcity/utils/change_planner_utils.py

```diff
@@ -96,6 +96,27 @@ def detect_conflicts_cp(sender, **kwargs):
     ).update(asset_conflict_asset_number=asset)
 
 
+def get_filtered_decommissioned_assets_for_cp(change_plan, data):
+    decommissioned_assets = DecommissionedAsset.objects.all()
+    decomissioned_assets_cp = AssetCP.objects.filter(
```

Comment:
Do we not have a DecommissionedAssetCP model?? Wild

Reply:
ah ha ha ...

Reply:
we elected to not have one.